CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 30.0.0
+
+- Removed: `stylelint` less than `15.0.0` from peer dependencies.
+- Removed: rules deprecated in `stylelint@15.0.0`.
```

Comment:
I know this is time consuming, but since the CHANGELOG of the [version 15](https://github.com/stylelint/stylelint/pull/6425/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR3) doesn't contain the list, could you either link the list here or list the rules directly?
I am asking because I almost didn't release how many rules you were removing; this needs to be explicit.

Reply:
That makes sense. I think adding a link to the [migration guide](https://github.com/stylelint/stylelint/blob/v15/docs/migration-guide/to-15.md) like this is enough because the guide describes the deprecated rules. Any thoughts?

```suggestion
- Removed: rules deprecated in `stylelint@15.0.0`. For details, see the [migration guide](https://github.com/stylelint/stylelint/blob/15.0.0/docs/migration-guide/to-15.md).
```

Reply:
This markdown file doesn't have the list so it's easy to miss. At the minimum it should contain the number of rules removed:
- Removed: **64** rules deprecated in `stylelint@15.0.0`. For details, see the [migration guide](https://github.com/stylelint/stylelint/blob/15.0.0/docs/migration-guide/to-15.md).

Reply:
Thanks, I've pushed the change afffa70.